examples/openvino/aot/README.md

```diff
@@ -0,0 +1,115 @@
+# **Model Export Script for Executorch**
+
+This script allows users to export deep learning models from various model suites (TIMM, Torchvision, Hugging Face) to a openvino backend using **Executorch**. Users can dynamically specify the model, input shape, and target device.
+
+
+## **Usage**
+
+### **Command Structure**
+```bash
+python aot_openvino_compiler.py --suite <MODEL_SUITE> --model <MODEL_NAME> --input_shape <INPUT_SHAPE> --device <DEVICE>
+```
+
+### **Arguments**
+- **`--suite`** (required):
+  Specifies the model suite to use.
+  Supported values:
+  - `timm` (e.g., VGG16, ResNet50)
+  - `torchvision` (e.g., resnet18, mobilenet_v2)
+  - `huggingface` (e.g., bert-base-uncased). NB: Quantization and validation is not supported yet.
+
+- **`--model`** (required):
+  Name of the model to export.
+  Examples:
+  - For `timm`: `vgg16`, `resnet50`
+  - For `torchvision`: `resnet18`, `mobilenet_v2`
+  - For `huggingface`: `bert-base-uncased`, `distilbert-base-uncased`
+
+- **`--input_shape`**(optional):
+  Input shape for the model. Provide this as a **list** or **tuple**.
+  Examples:
+  - `[1, 3, 224, 224]` (Zsh users: wrap in quotes)
+  - `(1, 3, 224, 224)`
+
+- **`--batch_size`** :
+  Batch size for the validation. Default batch_size == 1.
+  The dataset length must be evenly divisible by the batch size.
+
+- **`--quantize`** (optional):
+  Enable model quantization. --dataset argument is requred for the quantization. `huggingface` suite  does not supported yet.
+
+- **`--quantization_flow`** (optional):
+  Specifies the way to quantize torch.fx.GraphModule.
+  Supported values:
+  - `nncf`: `nncf quantize_pt2e` API (default)
+  - `pt2e`: torch ao quantization pipeline.
```

Comment:
what is the difference between these two? is nncf quantizer not compatible with pt2e flow?

Reply:
Thanks for the question. The pt2e quantization pipeline differs from nncf quantization in that the nncf quantization pipeline automatically applies algorithms to models before calibration, such as SmoothQuant, ChannelAligment, and after calibration, such as BiasCorrection.

OpenVINO quantizer is compatible with pt2e flow and taking into account the previous comments we will remove this argument.